tests/test_graph.py

```diff
@@ -275,9 +275,9 @@ def test_visualise_static_color(graph_constructor):
     @staticmethod
     def test_visualise_invalid_input(graph_constructor):
         """Test visualising a graph with invalid input."""
-        with pytest.raises(ValueError, match="must contain x"):
+        with pytest.raises(ValueError, match="must contain key `x`"):
             graph_constructor.visualise({})
-        with pytest.raises(ValueError, match="must contain edge_index"):
+        with pytest.raises(ValueError, match="must contain key `edge_index`"):
             graph_constructor.visualise({"x": []})
-        with pytest.raises(ValueError, match="must contain coords"):
+        with pytest.raises(ValueError, match="must contain key `coords`"):
```

Comment:
I think we should use `coordinates` instead of `coords` that's what I have done in #281

Reply:
It must be 'coords' here as that is what py-geometric expects it to be. 'Coordinates' will not work.

Reply:
@shaneahmed  I just realised that this is the one key which torch geometric does not require being a set key. Therefore, we could change this to be 'coordinates' if you like instead. The keys `x` and `edge_index` are set though.

Reply:
I would prefer consistency over spellings but if torch geometric doesn't require this then we spellings get the priority.